test/e2e/mcd_test.go

```diff
@@ -40,3 +43,50 @@ func TestMCDToken(t *testing.T) {
 		}
 	}
 }
+
+func TestSSHAccessedAnnotation(t *testing.T) {
+	cb, err := common.NewClientBuilder("")
+	if err != nil {
+		t.Errorf("%#v", err)
+	}
+	k := cb.KubeClientOrDie("test-ssh-accessed")
+
+	nodes, err := k.CoreV1().Nodes().List(metav1.ListOptions{})
+	if err != nil {
+		t.Errorf("failed to list nodes %v", err)
+	}
+
+	var (
+		nodeName string
+		nodeIP   string
+	)
+	for _, node := range nodes.Items {
+		for _, addr := range node.Status.Addresses {
+			// just pick a master with an external IP
+			if addr.Type == v1.NodeExternalIP {
+				nodeIP = addr.Address
+				nodeName = node.Name
+			}
+		}
+	}
+
+	cmd := exec.Command("ssh", "-oStrictHostKeyChecking=no", "core@"+nodeIP, "true")
+	if err := cmd.Run(); err != nil {
+		t.Errorf("error ssh'ing into a node")
+	}
+
+	err = wait.Poll(2*time.Second, 5*time.Minute, func() (bool, error) {
+		node, err := k.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
+		if err != nil {
+			return false, err
+		}
+		sshAnnotation, ok := node.ObjectMeta.Annotations["machineconfiguration.openshift.io/ssh"]
+		if !ok {
+			return false, nil
+		}
+		if sshAnnotation == "accessed" {
```

Comment:
It'd be even more convincing if it checked *before* doing `ssh` that the annotation was absent.

Reply:
let's do that

Reply:
done